src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineBranch.java

```diff
@@ -61,6 +62,19 @@ public boolean hasPhaseControlCapability() {
         return Collections.emptyList();
     }
 
+    @Override
+    public List<LfLimit> getLimits1(final LimitType type) {
+        if (type != LimitType.CURRENT) {
+            throw new UnsupportedOperationException(String.format("Getting %s limits is not supported on dangling line branch.", type.name()));
+        }
+        return getLimits1(type, danglingLine.getCurrentLimits());
```

Comment:
You are not able to get active power and apparent power limits on the side 1 of the dangling line?

Reply:
Actually I think I can, must have been confused by DanglingLine interface that only has a getCurrentLimits but inherits the rest from FlowsLimitsHolder.

Reply:
You think that we miss something in `powsybl-core`? User-friendy methods are missing?

Reply:
No, DanglineLineImpl gives the requested implementation for current, activepower and apparentpower. Dangling line interface just has a specific re declaration of getCurrentLimits which is curious. But all is there to support the three on openloadflow side.

Reply:
@MioRtia Do you have an opinion on that?

Reply:
Yes, sorry, it is a mistake because these methods existed before the introduction of the `FlowsLimitHolder` class... I will make a PR to delete them.